### ISSUE TITLE
Move to karma-runner organization

### ISSUE THREAD

**Issue by nickserv** [nickmccurdy/karma-edge-launcher#2]

**Comment by nickserv** [comment]
I'll wait on requesting this until I think the project is stable.

**Comment by Feytality** [comment]
Hey @nickmccurdy, just want to say your work is appreciated! I would just like to know if you plan on making a release any time soon?

**Comment by nickserv** [comment]
Thanks! This is essentially a wrapper around [edge-launcher](https://github.com/microsoftedge/edge-launcher) now, though in my experience it seems to be pretty stable.

I haven't been able to release this on npm yet. I'll take another look at publishing it and talk to Karma upstream developers, but for now you should be able to install it through GitHub with `npm install --save-dev nickmccurdy/karma-edge-launcher`. I wouldn't recommend using this on a production test suite, though.

**Comment by nickserv** [comment]
@Feytality This is now released on npm as `karma-edge-launcher`.

**Comment by dignifiedquire** [comment]
@nickmccurdy happy to take this into the karma organisation any time you feel it's ready.

**Comment by nickserv** [comment]
@dignifiedquire Thanks! The current status is deployed on npm, pre 1.0, with at least one known bug (Edge doesn't close after the tests finish unlike most launchers). I can help move it over if you're okay with having a somewhat experimental / in development plugin in the organization, otherwise we can wait until 1.0.

**Comment by dignifiedquire** [comment]
Happy to move it over now, just need you to give me admin access to this repo so I can move it. After that I will add you back as an admin on the other side

**Comment by nickserv** [comment]
Ah I see, it won't let me move to karma-runner without that. Do I have to add you as a collaborator, or is it something else?

**Comment by dignifiedquire** [comment]
@nickmccurdy I'm afraid I need moar permissions on here, I think it's called `Admin` or sth like that, such that I can access the `Settings` page.

**Comment by nickserv** [comment]
I wasn't able to add you as anything other than a collaborator. If that doesn't work, I think I would either need to join a karma-runner team or transfer this repository to your personal account so you could transfer it into karma-runner.

**Comment by nickserv** [comment]
Hey @dignifiedquire, are you still interested in this? If so, please let me know what I can do to help.

Attempting to transfer it myself still gives me a "You don’t have the permission to create repositories on karma-runner" error. Instead, would you like me to transfer this to your personal account since you'd have permission to then transfer it to karma-runner?

**Comment by nickserv** [comment]
I would like to try this again, could I please get repository creation access to @karma-runner?

**Comment by mgol** [comment]
@nickmccurdy Repository creation access is not needed for that, someone just needs to create a repo for you and add you to the list of people with write access to that repo.

I'll ask around.

**Comment by dignifiedquire** [comment]
I have created https://github.com/karma-runner/karma-edge-launcher and gave @nickmccurdy full access. Let me know if that helps

**Comment by nickserv** [comment]
Thanks for the repository!

Unfortunately it seems like I still can't transfer my repository through GitHub, I get the following error:
> karma-runner/karma-edge-launcher already exists and You don’t have the permission to create repositories on karma-runner`

Do you think someone in @karma-runner could initiate the transfer instead of me? If this isn't possible I will transfer the commits, open issues, and hooks manually, but it would be nice to maintain all the history with a transfer.

**Comment by aligneddev** [comment]
It looks like this has stalled. Any updates?

It would be good to nice to have this moved into the karma-runner organization. We are starting to put it into our build after getting a Windows 10 build agent.

**Comment by nickmccurdy** [comment]
Right now I have the ability to push all the code to the @karma-runner repo, but because I'm not a team member I can't transfer the repo with issues and pull requests in tact. Since there's only one each (besides this issue), I'll try to do this manually. Should be transferred soon.

**Comment by nickserv** [comment]
Moved to [karma-runner/karma-edge-launcher](https://github.com/karma-runner/karma-edge-launcher) and released under `karma-edge-launcher@0.4.2`. Please update your GitHub watching, forks, bookmarks, and links as appropriate.
